code/game/machinery/computer/orders/order_computer/mining_order.dm

```diff
@@ -1,15 +1,19 @@
+#define MINING_SHIPPING_MULTIPLIER 0.65
+#define GET_MINING_SHIPPING_MULTIPLIER(cost) round(cost * MINING_SHIPPING_MULTIPLIER,5)
+
 /obj/machinery/computer/order_console/mining
 	name = "mining equipment order console"
 	desc = "An equipment shop for miners, points collected at an ore redemption machine can be spent here."
 	icon = 'icons/obj/machines/mining_machines.dmi'
 	icon_state = "mining"
 	icon_keyboard = null
 	icon_screen = null
+	cargo_cheaper = TRUE
```

Comment:
I feel like I'm repeating myself, I still do not agree with separating express cost multiplier and ``cargo_cheaper``, two separate systems one meant to make items more expensive when using cargo, and another meant to make items cheaper using cargo (that uses defines instead of the pre-existing vars) just confuses the whole system, the same thing you've done here can just be done by directly lowering the prices of the items which is what I've been saying to do from the start. It does the same thing at the end of the day, it's just that you're overcomplicating the machine for no reason.

Reply:
this variable literally just changes the tooltips

Reply:
Again, use ``express_cost_multiplier``, the point of the original change was to make **the chef and mining consoles use the same system**. You want to show express cost in the UI, sure, but now it's not shown in the Chef's version. You keep trying to do these workarounds to constantly go around me telling you to just **use the system that is there instead of making a new one and snowflaking it in**.

Reply:
" the point of the original change was to make the chef and mining consoles use the same system." (said system does not fit mining at all) goober

Reply:
you told me not to use express_cost_multiplier you goober

Reply:
also check the new commits wackass

Reply:
thank you for those